### ISSUE TITLE
status breakdown

### ISSUE THREAD

**Issue by zuoshouzz** [wazuh/wazuh-kibana-app#376]
<img width="1031" alt="wx20180417-152121 2x" src="https://user-images.githubusercontent.com/13675540/38854649-bd40a956-4253-11e8-9e48-c553b99a2019.png">

**Comment by JuanjiJG** [comment]
Hello @zuoshouzz,

Let's see if we can help you with this problem.

Did the error appear doing something in particular, or was it suddenly?

We can also see the status log and the log files to find any possible errors.
```bash
systemctl status elasticsearch -l
journalctl -xe | grep elasticsearch
```

Try to restart the Elasticsearch server and see if you can perform the following CURL call:
```bash
systemctl restart elasticsearch
(wait 20 seconds to completely restart the ES server)
```

```bash
curl localhost:9200?pretty
```

Expected output:
```json
{
  "name" : "4ltE1w9",
  "cluster_name" : "elasticsearch",
  "cluster_uuid" : "8KpXmQCOT5yB89I7dOGk_w",
  "version" : {
    "number" : "6.2.3",
    "build_hash" : "c59ff00",
    "build_date" : "2018-03-13T10:06:29.741383Z",
    "build_snapshot" : false,
    "lucene_version" : "7.2.1",
    "minimum_wire_compatibility_version" : "5.6.0",
    "minimum_index_compatibility_version" : "5.0.0"
  },
  "tagline" : "You Know, for Search"
}
```

Thanks for your patience.

Regards,
Juanjo

**Comment by zuoshouzz** [comment]
![image](https://user-images.githubusercontent.com/13675540/38856036-de9f7970-4257-11e8-9608-d03ac65f35ac.png)
![image](https://user-images.githubusercontent.com/13675540/38856052-eb27602c-4257-11e8-9099-d795860c51da.png)

**Comment by JuanjiJG** [comment]
Hello again @zuoshouzz,

I can see some Java garbage collection warnings on your screenshots. Let's see your current RAM status. Please, execute this command and paste here the output:
```bash
free -h
```

Thanks and regards,
Juanjo

**Comment by zuoshouzz** [comment]
![image](https://user-images.githubusercontent.com/13675540/38857877-e9b76ad4-425c-11e8-9478-7dac8eacf0a9.png)

**Comment by JuanjiJG** [comment]
Hello again @zuoshouzz, and sorry for the late response.

The problem is not your available RAM, but it seems there's a problem with the heap memory used by the Java garbage collector.

This problem looks more like an Elasticsearch problem, so in addition to our help, I recommend you going to the [Elastic discussion forums](https://discuss.elastic.co/) and post a thread describing your problem, so the Elastic team can help you with this.

Are you using a cluster for Elasticsearch? Let's check out its current state:
```
curl -X GET "localhost:9200/_cluster/stats?human&pretty"
```

Please, paste the logs in text format, not just logs images. That will help us to better locate some useful information.

Thanks and regards,
Juanjo

**Comment by JuanjiJG** [comment]
Hello again @zuoshouzz,

It's been more than a month since the last response to this issue, so I'll proceed to close it.

Don't hesitate to open a new one if you still have questions or problems using Wazuh.

Regards,
Juanjo

**Comment by Apurva09** [comment]
I'm getting status red on kibana dashboard and unable to create index .kibana
while giving this command
 curl -X GET "localhost:9200/_cluster/stats?human&pretty"
 
please help me with this issue

 {
  "_nodes" : {
    "total" : 1,
    "successful" : 1,
    "failed" : 0
  },
  "cluster_name" : "my-application",
  "timestamp" : 1565173814026,
  "status" : "red",
  "indices" : {
    "count" : 0,
    "shards" : { },
    "docs" : {
      "count" : 0,
      "deleted" : 0
    },
    "store" : {
      "size" : "0b",
      "size_in_bytes" : 0
    },
    "fielddata" : {
      "memory_size" : "0b",
      "memory_size_in_bytes" : 0,
      "evictions" : 0
    },
    "query_cache" : {
      "memory_size" : "0b",
      "memory_size_in_bytes" : 0,
      "total_count" : 0,
      "hit_count" : 0,
      "miss_count" : 0,
      "cache_size" : 0,
      "cache_count" : 0,
      "evictions" : 0
    },
    "completion" : {
      "size" : "0b",
      "size_in_bytes" : 0
    },
    "segments" : {
      "count" : 0,
      "memory" : "0b",
      "memory_in_bytes" : 0,
      "terms_memory" : "0b",
      "terms_memory_in_bytes" : 0,
      "stored_fields_memory" : "0b",
      "stored_fields_memory_in_bytes" : 0,
      "term_vectors_memory" : "0b",
      "term_vectors_memory_in_bytes" : 0,
      "norms_memory" : "0b",
      "norms_memory_in_bytes" : 0,
      "points_memory" : "0b",
      "points_memory_in_bytes" : 0,
      "doc_values_memory" : "0b",
      "doc_values_memory_in_bytes" : 0,
      "index_writer_memory" : "0b",
      "index_writer_memory_in_bytes" : 0,
      "version_map_memory" : "0b",
      "version_map_memory_in_bytes" : 0,
      "fixed_bit_set" : "0b",
      "fixed_bit_set_memory_in_bytes" : 0,
      "max_unsafe_auto_id_timestamp" : -9223372036854775808,
      "file_sizes" : { }
    }
  },
  "nodes" : {
    "count" : {
      "total" : 1,
      "data" : 1,
      "coordinating_only" : 0,
      "master" : 1,
      "ingest" : 1
    },
    "versions" : [
      "6.0.1"
    ],
    "os" : {
      "available_processors" : 40,
      "allocated_processors" : 40,
      "names" : [
        {
          "name" : "Linux",
          "count" : 1
        }
      ],
      "mem" : {
        "total" : "62.8gb",
        "total_in_bytes" : 67519180800,
        "free" : "54.1gb",
        "free_in_bytes" : 58169155584,
        "used" : "8.7gb",
        "used_in_bytes" : 9350025216,
        "free_percent" : 86,
        "used_percent" : 14
      }
    },
    "process" : {
      "cpu" : {
        "percent" : 0
      },
      "open_file_descriptors" : {
        "min" : 819,
        "max" : 819,
        "avg" : 819
      }
    },
    "jvm" : {
      "max_uptime" : "2h",
      "max_uptime_in_millis" : 7453431,
      "versions" : [
        {
          "version" : "1.8.0_71",
          "vm_name" : "Java HotSpot(TM) 64-Bit Server VM",
          "vm_version" : "25.71-b15",
          "vm_vendor" : "Oracle Corporation",
          "count" : 1
        }
      ],
      "mem" : {
        "heap_used" : "230.1mb",
        "heap_used_in_bytes" : 241279560,
        "heap_max" : "989.8mb",
        "heap_max_in_bytes" : 1037959168
      },
      "threads" : 104
    },
    "fs" : {
      "total" : "295.1gb",
      "total_in_bytes" : 316934193152,
      "free" : "284.7gb",
      "free_in_bytes" : 305699201024,
      "available" : "269.7gb",
      "available_in_bytes" : 289593073664
    },
    "plugins" : [ ],
    "network_types" : {
      "transport_types" : {
        "netty4" : 1
      },
      "http_types" : {
        "netty4" : 1
      }
    }
  }
}

**Comment by Apurva09** [comment]
my elastic search output on server using curl command

[root@NVMBD1CDT070D00 ~]# curl -X GET "10.135.140.118:9200"
{
  "name" : "NVMBD1CDT070D00",
  "cluster_name" : "my-application",
  "cluster_uuid" : "E1Xv7fiDR7-ZUWIygJEFjQ",
  "version" : {
    "number" : "6.0.1",
    "build_hash" : "601be4a",
    "build_date" : "2017-12-04T09:29:09.525Z",
    "build_snapshot" : false,
    "lucene_version" : "7.0.1",
    "minimum_wire_compatibility_version" : "5.6.0",
    "minimum_index_compatibility_version" : "5.0.0"
  },
  "tagline" : "You Know, for Search"
}
